tmk_core/common/sync_timer.h

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2011 Jun Wako <wakojun@gmail.com>
```

Comment:
```suggestion
Copyright 2020
```
Should put your name here, I guess.

Reply:
fixed